src/main/java/de/qabel/desktop/ui/accounting/menuqr/MenuQRController.java

```diff
@@ -0,0 +1,68 @@
+package de.qabel.desktop.ui.accounting.menuqr;
+
+import de.qabel.core.config.Identity;
+import de.qabel.desktop.ui.AbstractController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeView;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import org.controlsfx.control.PopOver;
+
+import javax.inject.Inject;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class MenuQRController extends AbstractController implements Initializable {
```

Comment:
It the menu really QR specific? It's more like an IdentityContextMenu or something like that (all methods are related to the Identity but not all are related to the QRCode)